tekton/ci/jobs/tekton-catlin-lint.yaml

```diff
@@ -34,11 +37,33 @@ spec:
       image: gcr.io/tekton-releases/dogfooding/catlin:latest
       workingDir: $(resources.inputs.source.path)
       script: |
+        set +e
```

Comment:
I just realized that the job is going to always pass with this `set +e`. Will fix this tomorrow :sweat_smile: